python/rapidsmpf/rapidsmpf/streaming/chunks/arbitrary.pyx

```diff
@@ -0,0 +1,154 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES.
+# SPDX-License-Identifier: Apache-2.0
+
+from cpython.object cimport PyObject
+from cpython.ref cimport Py_DECREF, Py_INCREF
+from cython.operator cimport dereference as deref
+from libc.stdint cimport uint64_t
+from libcpp.memory cimport make_unique, unique_ptr
+from libcpp.utility cimport move
+
+from rapidsmpf.streaming.chunks.utils cimport py_deleter
+from rapidsmpf.streaming.core.message cimport Message, cpp_Message
+
+
+cdef extern from * nogil:
+    """
+    namespace {
+    rapidsmpf::streaming::Message to_message(
+        std::uint64_t sequence_number,
+        std::unique_ptr<rapidsmpf::streaming::OwningWrapper> obj
+    ) {
+        return rapidsmpf::streaming::Message(sequence_number, std::move(obj));
+    }
+    }
+    """
+    cpp_Message cpp_to_message"to_message"(
+        uint64_t, unique_ptr[cpp_OwningWrapper]
+    ) except +
+
+
+cdef class ArbitraryChunk:
+    def __init__(self, object obj):
```

Comment:
```suggestion
    def __init__(self, object obj not None):
```

Reply:
`None` is a valid payload (ok, kind of a dumb one, but still).